<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectToServerDialog</class>
 <widget class="QDialog" name="ConnectToServerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to server</string>
  </property>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>331</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect!</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="guestOrAccountTabs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabGuest">
    <attribute name="title">
     <string>I'm a guest</string>
    </attribute>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>311</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelNIckname">
        <property name="text">
         <string>Nickname</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineNickname"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabAccount">
    <attribute name="title">
     <string>I already have an account</string>
    </attribute>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>311</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelUsername">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineUsername"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="linePassword">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineServer">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>281</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelServer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>141</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Server</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
